packages/circuit-ui/components/Popover/Popover.spec.tsx

```diff
@@ -215,9 +210,35 @@ describe('Popover', () => {
       expect(baseProps.onToggle).toHaveBeenCalledTimes(1);
     });
 
-    /**
-     * Accessibility tests.
-     */
+    it('should move focus to the first popover item after opening', () => {
+      const { getAllByRole, rerender } = renderPopover({
+        ...baseProps,
+        isOpen: false,
+      });
+
+      act(() => {
+        rerender(<Popover {...baseProps} isOpen />);
+      });
+
+      const popoverItems = getAllByRole('menuitem');
+
+      expect(document.activeElement).toEqual(popoverItems[0]);
```

Comment:
👀 didn't know this one!We could also use [`toHaveFocus()`](https://github.com/testing-library/jest-dom#tohavefocus)

Reply:
Oh, that's much better!

Reply:
Changed in 61d11cf.